pkg/controller/jaeger/upgrade.go

```diff
@@ -18,6 +18,11 @@ func (r *ReconcileJaeger) applyUpgrades(ctx context.Context, jaeger v1.Jaeger) (
 
 	currentVersions := version.Get()
 
+	err := upgrade.AddLabelsToExistingServiceAccounts(ctx, r.rClient, jaeger)
```

Comment:
Do we want to add labels to all the upgrades?

Reply:
As per my opinion yes because these labels are being checked from version `0.11.0` and the latest version is `1.46.0`. I don't think any one will be using version before a stable one which should be `1.0.0`. So I think this should apply all the future releases. But still just to confirm I have already asked this with maintainer to confirm my assumption/opinion.